Apps/GatewayApi/test/unit/Services.Test/DelegateServiceTests.cs

```diff
@@ -86,12 +86,12 @@ public static IEnumerable<object[]> TestCases()
         public async Task CreateDelegateInvitation(string email, string nickname, int daysToAdd, HashSet<DataSource> dataSources, bool success)
```

Comment:
Name and documentation of test method should be updated as well.

Reply:
Changed